### DISCUSSION TITLE
'Layerize' b&w image

### DISCUSSION THREAD

**Discussion by ovosodo** [ImageMagick/ImageMagick#6369] [Digital Image Processing]
Hello,
I miss from IM from a long time, can't use my old nick [myspacee] so, hello again.

I know that Photo editing programs can reach this effect :
<a href="https://ibb.co/zmsVp48"><img src="https://i.ibb.co/K56xdrN/asd.jpg" alt="asd" border="0"></a>

But is there any way to do this using IM ?

Thank you for your time,
M.

PS: english is not my first language, can you suggest english words to describe this image effect ?

**Comment by fmw42** [comment]
Please show your original image before the effect is added.

**Comment by fmw42** [comment]
Is this what you want?

![face](https://github.com/ImageMagick/ImageMagick/assets/13458218/fe071762-7db7-4b50-8272-305031af7d7e)

```
n=6
convert face.png -colorspace gray -blur 0x2 -fx "int($n*u+0.5)/$n" x.png

```

![x](https://github.com/ImageMagick/ImageMagick/assets/13458218/f740d9bc-f75f-4737-8e4b-3869946fc57a)

Adjust the blur value for smoothness.  Adjust n for the number of contours.

**Comment by snibgo** [comment]
See also `-posterize`: https://imagemagick.org/script/command-line-options.php#posterize
